src/components/input/input.html

```diff
@@ -0,0 +1,178 @@
+                    <path
+                        d="M256 160c-52.9 0-96 43.1-96 96s43.1 96 96 96 96-43.1 96-96-43.1-96-96-96zm246.4 80.5l-94.7-47.3 33.5-100.4c4.5-13.6-8.4-26.5-21.9-21.9l-100.4 33.5-47.4-94.8c-6.4-12.8-24.6-12.8-31 0l-47.3 94.7L92.7 70.8c-13.6-4.5-26.5 8.4-21.9 21.9l33.5 100.4-94.7 47.4c-12.8 6.4-12.8 24.6 0 31l94.7 47.3-33.5 100.5c-4.5 13.6 8.4 26.5 21.9 21.9l100.4-33.5 47.3 94.7c6.4 12.8 24.6 12.8 31 0l47.3-94.7 100.4 33.5c13.6 4.5 26.5-8.4 21.9-21.9l-33.5-100.4 94.7-47.3c13-6.5 13-24.7.2-31.1zm-155.9 106c-49.9 49.9-131.1 49.9-181 0-49.9-49.9-49.9-131.1 0-181 49.9-49.9 131.1-49.9 181 0 49.9 49.9 49.9 131.1 0 181z">
+                    </path>
+                </svg>
+            </div>
+        </div>
+    </header>
+
+    <!-- aside start -->
+    <aside>
+        <!-- Getting Started -->
+        <h3>Getting Started</h3>
+        <a href="">Introduction</a>
+        <a href="">Typography</a>
+
+        <!-- components -->
+        <h3>Components</h4>
+            <a href="../alert/alert.html">Alert</a>
+            <a href="../avatar/avatar.html">Avatar</a>
+            <a href="../badge/badge.html">Badge</a>
+            <a href="../button/button.html">Button</a>
+            <a  href="../card/card.html">Card</a>
+            <a href="../image/image.html">Image</a>
+            <a class="active-tab" href="">Input</a>
+            <a href="">Form</a>
+            <a href="">Snackbar</a>
+    </aside>
+
+    <!-- main -->
+    <main>
+        <h1>Input</h1>
+        <p class="body-l"></p>
+
+        <!-- inputs -->
+        <div class="component-desc">
+            <h3>Text Input</h3>
+            <p>By default artsy ui inputs are outline styled, to change to standard style input add class <span class="highlight-text">input-std</span> to the input.<br>
+            To mark a input as req just add class <span class="highlight-text">input-req</span> to the div. </p>
+        </div>
+        <div class="component-container ">
+            <div class="artsy-input" >
+                <span class="input-label">First Name</span>
+                <input  type="text"/>
+            </div>
+            <div class="artsy-input input-req" >
+                <span class="input-label">First Name</span>
+                <input type="text"/>
+            </div>
+            
+            <div class="artsy-input" >
+                <span class="input-label">First Name</span>
+                <input class="input-std" type="text"/>
+            </div>
+        </div>
+
+        <!-- input w/icon -->
+        <div class="component-desc">
+            <h3>Input with Icon</h3>
+            <p>Add a icon and give class <span class="highlight-text">icon-input</span> to the div. </p>
+        </div>
+        <div class="component-container ">
+            <div class="artsy-input icon-input" >
+                <span class="input-label">search</span>
+                <i class="fas fa-search"></i>
```

Comment:
the search icon would look a lot better in the right end of the search field.
Also, your input fields look really good.